pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
A JIRA ticket sounds like a good idea to me.  I think the question that needs to be answered here is if we are going to run into other trouble by never cleaning out `test_root/staticfiles` before tests.